backend/src/hatchling/metadata/core.py

```diff
@@ -1357,6 +1358,23 @@ def dynamic(self) -> list[str]:
 
         return self._dynamic
 
+    @property
+    def sbom_files(self) -> list[str]:
+        """
+        https://peps.python.org/pep-0770/
+        """
+        sbom_files = self.config.get("sbom-files", [])
```

Comment:
This isn't the right field, it was `dist-info.files.sbom` provisionally but that was actually [removed](https://github.com/python/peps/pull/4321) and is waiting on a standalone PEP. I asked about that [here](https://discuss.python.org/t/76308/113).

Until there is a standard we cannot read from the project table. Instead, we should offer configuration as both wheel [build data](https://hatch.pypa.io/latest/plugins/builder/wheel/#build-data) and [options](https://hatch.pypa.io/latest/plugins/builder/wheel/#options). I would have imagined that the former would be the primary use case as such files are typically generated at build time but then the (eventual) spec is supposedly about a static field/location, so we also should support options.

Reply:
Moved to the corrected location and removed this from core metadata.